msdsn/conn_str.go

```diff
@@ -72,6 +74,15 @@ type Config struct {
 	ConnTimeout time.Duration // Use context for timeouts.
 	KeepAlive   time.Duration // Leave at default.
 	PacketSize  uint16
+
+	// Kerberos authentication fields
+
+	Krb5ConfFile      string
+	KrbCache          string
+	Realm             string
+	Initkrbwithkeytab string
+	Keytabfile        string
```

Comment:
This should be the actual keytab file content, or even better the parsed content. See Encryption and TLSConfig
 fields for examples of what is needed for these two file properties.